scripts/linkValidator.test.md

```diff
@@ -0,0 +1,49 @@
+# Tests for linkValidator.js
```

Comment:
maybe consider moving this file to `/scripts/tests/`?

Reply:
great suggestion, done